moor_generator/lib/src/model/sql_query.dart

```diff
@@ -568,7 +568,7 @@ class ExpressionDartPlaceholderType extends DartPlaceholderType {
     if (columnType == null) return 'Expression';
 
     final dartType = dartTypeNames[columnType];
-    return 'Expression<$dartType>';
+    return 'Expression<$dartType?>';
```

Comment:
Moor must still support projects that haven't migrated to null-safety yet, so we can't unconditionally use null-safe syntax.

```suggestion
    // Use a nullable type because most columns are currently nullable in
    // the Dart api.
    return 'Expression<${options.nullableType(dartType)}>';
```

Reply:
> I agree that making the expression nullable is the way to go. It's not perfect, but with the current column nullability in Dart there's no other option really.

> Also, could you change the base to `develop`? That makes it easier for me to combine this with the current state and release everything together, thanks!
done

> Moor must still support projects that haven't migrated to null-safety yet, so we can't unconditionally use null-safe syntax.
done